lua/wikis/commons/Array.lua

```diff
@@ -530,7 +530,7 @@ Array.reduce({2, 3, 5}, pow)
 ---@generic T, V
 ---@param array T[]
 ---@param operator fun(aggregate: V, arrayValue: T): V
----@param initialValue V
+---@param initialValue V?
```

Comment:
I'm not sure if this actually is needed, I think the nil possiblity gets accounted into the general V?

Reply:
vscode yelled at me in several modules, adding this ? shut it up